docker/Dockerfile

```diff
@@ -1,5 +1,6 @@
-FROM paddlepaddle/paddle
```

Comment:
Sorry, I do not add `Dockerfile` in `.gitignore` in the early PR, we can use `build_docker.sh` to build a Docker image, so could you add `Dockerfile` in `.gitignore` in this PR, thanks!